crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -533,7 +554,10 @@ impl ViewportBlueprint {
                 }
 
                 match container {
-                    egui_tiles::Container::Tabs(_) => {}
+                    egui_tiles::Container::Tabs(tabs) => {
+                        // TODO(abey79): Need to add active tab to the blueprint spec
```

Comment:
There is a lot of new `TODO(abey79)` added in commits by @jleibs in this PR, which I find weird.

I interpret the name in the TODO as the signature of who wrote the TODO (i.e. who to talk to for clarification about it), not the name of whom is supposed to do todo.

But maybe you collaborated on these commits? Or are you having a personality crisis? 😬

Reply:
Yeah -- a bit confusing there. Many of these came up in the list of things we were talking through when pairing on the initial implementation.